File: Cocktail/src/main/kotlin/org/programmers/cocktail/search/dto/CommentsTO.java
package org.programmers.cocktail.search.dto;

import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class CommentsTO {
    private Long id;
    private String content;
    private Long userId;
    private String userName;
    private Long cocktailId;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;

}

File: Cocktail/src/main/kotlin/org/programmers/cocktail/search/dto/CommentsTO.kt
package org.programmers.cocktail.search.dto

import java.time.LocalDateTime

data class CommentsTO (
    var id: Long? = null,
    var content: String? = null,
    var userId: Long? = null,
    var userName: String? = null,
    var cocktailId: Long? = null,
    var createdAt: LocalDateTime? = null,
    var updatedAt: LocalDateTime? = null
)
